cmd/compose/up.go

```diff
@@ -126,6 +127,7 @@ func upCommand(p *projectOptions, backend api.Service) *cobra.Command {
 	flags.BoolVar(&up.cascadeStop, "abort-on-container-exit", false, "Stops all containers if any container was stopped. Incompatible with -d")
 	flags.StringVar(&up.exitCodeFrom, "exit-code-from", "", "Return the exit code of the selected service container. Implies --abort-on-container-exit")
 	flags.IntVarP(&create.timeout, "timeout", "t", 10, "Use this timeout in seconds for container shutdown when attached or when containers are already running.")
+	flags.BoolVar(&up.timestamp, "timestamp", false, "Show timestamps.")
```

Comment:
For consistency with `compose logs` & `docker logs`, looks like this should be `timestamps` (plural)

https://github.com/docker/cli/blob/990674901b4fdd0bd2d3f91aa84ad3efe225f6fb/cli/command/container/logs.go#L49

https://github.com/docker/compose/blob/0368f190303b28e142f983257171a6f554749313/cmd/compose/logs.go#L60